### PR TITLE
Hfa 47 - feat(storage): add repository for photo storage

### PR DESCRIPTION
**Pull request by trofoSK**

### CODE REVIEW

File: src/main/java/sk/hfa/databases/projects/domains/repositories/FileSystemRepository.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        Path newFile = Paths.get(RESOURCES_DIR + new Date().getTime() + "-" + imageName);
+        Files.createDirectories(newFile.getParent());
+        Files.write(newFile, content);
+        return newFile.toAbsolutePath().toString();
+    }
+
+    public FileSystemResource findInFileSystem(String location) {
+        try {
+            return new FileSystemResource(Paths.get(location));
+        } catch (Exception e) {
```
Comment (RandomUserIK) [inline src/main/java/sk/hfa/databases/projects/domains/repositories/FileSystemRepository.java]: tu sa neda nejaka konkretnejsia vynimka odchytit?

File: src/main/java/sk/hfa/databases/projects/services/IndividualProjectService.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    @Override
+    public IndividualProject findIndividualProjectById(Long id) {
+        Optional<IndividualProject> post = individualProjectRepository.findById(id);
+        return post.orElse(null);
+    }
+
+    @Override
+    public IndividualProject findIndividualProjectByProjectName(String projectName) {
+        Optional<IndividualProject> post = individualProjectRepository.findByProjectName(projectName);
```
Comment (RandomUserIK) [inline src/main/java/sk/hfa/databases/projects/services/IndividualProjectService.java]: mozes rovno dat `return individualProjectRepository.findByProjectName(projectName).orElse(null)`

File: src/main/java/sk/hfa/databases/projects/services/IndividualProjectService.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    private final IndividualProjectRepository individualProjectRepository;
+
+    public IndividualProjectService(IndividualProjectRepository individualProjectRepository) {
+        this.individualProjectRepository = individualProjectRepository;
+    }
+
+    @Override
+    public IndividualProject findIndividualProjectById(Long id) {
+        Optional<IndividualProject> post = individualProjectRepository.findById(id);
```
Comment (RandomUserIK) [inline src/main/java/sk/hfa/databases/projects/services/IndividualProjectService.java]: mozes rovno dat return `individualProjectRepository.findById(projectName).orElse(null)`

File: src/main/java/sk/hfa/databases/projects/domains/TextSections.java [hunk 4]
```diff
@@ -0,0 +1,17 @@
+package sk.hfa.databases.projects.domains;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Data
+@Entity
+public class TextSections {
```
Comment (RandomUserIK) [inline src/main/java/sk/hfa/databases/projects/domains/TextSections.java]: tu by som zmenil mozno nazov na `TextSection` kedze sa jedna o entitu a potom mas pekne v IndividualProject triede `List<TextSection> textSections`

**Review by RandomUserIK — CHANGES_REQUESTED** [review]

**Review by RandomUserIK — APPROVED** [review]

**Review by MartinMiklovic — APPROVED** [review]

**Pull request merged**